deploy/01-chaos-operator.yaml

```diff
@@ -19,9 +19,9 @@ metadata:
   labels:
     name: litmus
 rules:
-- apiGroups: ["","apps","batch","litmuschaos.io"]
-  resources: ["pods","jobs","deployments","daemonsets","statefulsets","events","configmaps","services","chaosengines","chaosexperiments","chaosresults"]
-  verbs: ["get","create","update","patch","delete","list","watch","deletecollection"]
+  - apiGroups: ["","apps","batch","litmuschaos.io","openshift.io"]
+    resources: ["pods","jobs","deployments","daemonsets","replicasets","deploymentconfigs","statefulsets","events","configmaps","services","chaosengines","chaosexperiments","chaosresults"]
```

Comment:
Do we need replicasets ?

Reply:
I suppose we need replication controllers in openshift context